Mon Jun 18 11:29:49 2007  Nobuyoshi Nakada  <nobu@ruby-lang.org>

	* intern.h, ext/thread/thread.c: moved prototype of rb_thread_status()
	  to get rid of error in C++.  [ruby-list:43615]

Index: intern.h
===================================================================
--- intern.h	(リビジョン 12566)
+++ intern.h	(リビジョン 12567)
@@ -204,7 +204,6 @@
 void rb_thread_polling _((void));
 void rb_thread_sleep _((int));
 void rb_thread_sleep_forever _((void));
-enum rb_thread_status rb_thread_status _((VALUE));
 VALUE rb_thread_stop _((void));
 VALUE rb_thread_wakeup _((VALUE));
 VALUE rb_thread_wakeup_alive _((VALUE));
Index: ext/thread/thread.c
===================================================================
--- ext/thread/thread.c	(リビジョン 12566)
+++ ext/thread/thread.c	(リビジョン 12567)
@@ -14,6 +14,8 @@
 #include <rubysig.h>
 #include <node.h>
 
+enum rb_thread_status rb_thread_status _((VALUE));
+
 static VALUE rb_cMutex;
 static VALUE rb_cConditionVariable;
 static VALUE rb_cQueue;
